docs/rest.md

```diff
@@ -540,3 +540,33 @@ Get an icon from a snap installed on the system. The response will be the raw
 contents of the icon file; the content-type will be set accordingly.
 
 This fetches the icon from the package itself.
+
+## /1.0/capabilities
+### GET
+
+* Description: List of snappy capabilities
+* Authorization: guest
+* Operation: sync
+* Return: information about all of the snappy capabilities 
```

Comment:
This won't return a list of all _snappy_ capabilities. This is related to the note on the documentation string early in this review.